src/main/res/layout/view_server_file_item.xml

```diff
@@ -18,19 +18,19 @@
   -->
 
 <androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    style="@style/Widget.MaterialComponents.CardView"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    xmlns:tools="http://schemas.android.com/tools"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:background="?android:attr/selectableItemBackground"
+    android:layout_margin="4dp"
     android:clickable="true"
     android:focusable="true"
-    style="@style/Widget.MaterialComponents.CardView"
+    android:foreground="?android:attr/selectableItemBackgroundBorderless"
```

Comment:
Where is the `android:background` attribute? We don't need that anymore?

Reply:
No, it is not required from API 21. We can instead set `android:foreground="?android:attr/selectableItemBackgroundBorderless"`